Training_25/Program.cs

```diff
@@ -3,11 +3,40 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program calculates the LCM and GCD of the given input numbers.
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
+using static System.Math;
 namespace Training_25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      int a = ReadValidInt ("Enter the first number: ");
+      int b = ReadValidInt ("Enter the second number: ");
+      WriteLine ($"LCM:{LCM (a, b)}");
+      WriteLine ($"GCD:{GCD (a, b)}");

```

Comment:
```suggestion
      WriteLine ($"GCD: {GCD (a, b)}");
```
There should always be a space after colon.